packages/nc-gui/components/template/Editor.vue

```diff
@@ -872,7 +869,7 @@ function handleUIDTChange(column, table) {
                       <a-select-option v-for="(option, i) of uiTypeOptions" :key="i" :value="option.value">
                         <a-tooltip placement="right">
                           <template v-if="isSelectDisabled(option.label, table.columns[record.key]?._disableSelect)" #title>
-                            The field is too large to be converted to {{ option.label }}
+                            {{ $t('msg.tooLargeField') }} {{ option.label }}
```

Comment:
Better to use i18n params

<img width="513" alt="image" src="https://github.com/nocodb/nocodb/assets/35809690/41ed52e8-80bc-443d-aaf7-7e91d009c8e9">